joomla/joomla-cms
# [4.0] Improve backend edit account

Pull Request for Issue # .

### Summary of Changes
PR https://github.com/joomla/joomla-cms/pull/32771 allows users without core.manage permission to edit their own accounts directly via com_users instead of via view profile of com_admin (view profile later was removed via PR https://github.com/joomla/joomla-cms/pull/32933)

However, it causes some changes when a manager edit his account:

- He can now change username even if **Change Username** parameter in com_users set to No
- He can see and changes some settings which he should not be allowed to change like Receiving System Emails, User Status...

This PR fixes that behavior. Basically, for users without core.manager permission, editing his account (now via com_users) should work the same as before (via com_admin)

### Testing Instructions

1. Apply Patch
2. Login to administrator area of your site using a manager user account
3. Access to **Edit Account** link from User Menu at the top right
4. Before patch:
- You can change username (when **Change Username** parameter in com_users set to **No** - default Joomla 4 setting)
- You can see some fields: Last Reset Date, Password Reset Count, Receive System Emails, User Status
5. After patch:
- You are not allowed to change username when **Change Username** parameter in com_users set to **No** 
- You won't see the fields Last Reset Date, Password Reset Count, Receive System Emails, User Status anymore
